PocketKit/Sources/PocketKit/MyList/ItemsList/ItemsListViewController.swift

```diff
@@ -252,7 +253,8 @@ class ItemsListViewController<ViewModel: ItemsListViewModel>: UIViewController,
             shareAction: model.shareAction(for: objectID),
             favoriteAction: model.favoriteAction(for: objectID),
             overflowActions: model.overflowActions(for: objectID),
-            filterByTagAction: model.filterByTagAction()
+            filterByTagAction: model.filterByTagAction(),
+            trackOverflow: model.trackOverflow(for: objectID)
```

Comment:
might want to refactor how we are utilizing the model with the cell, but decided to keep the pattern consistent and think about a refactor on a later date - let me know if any thoughts!